gdg_events/google_io_extended.md

```diff
@@ -2,6 +2,16 @@
 
 ![](https://events.google.com/io2015/images/io15-color.png)
 
+Every year, Google hosts developers around world in a software developer-focused conference held in San Francisco, California to showcase innovations in areas like Android mobile development, Modern front-end tools, infrastructure & DevOps, wearables, phones etc. 
+
+Google I/O features highly technical, in-depth sessions focused on building web, mobile, and enterprise applications with Google and open web technologies such as Android, Chrome, Chrome OS, Google APIs, Google Web Toolkit, App Engine, and more.
+
+As interesting as it sounds, many developers around the world feel left out as they are unable to attend the Google I/O event every year. Students and Industry Experts are always vying to get a glimpse of this much awaited event, but in vain.
```

Comment:
For lines 9 and 10, possibly combine them to sound as follows:
+To extend this to developers who are unable to attend the event on-site, and reach out to various communities like the Google Developer Groups, Google Business Groups, and others, an event known `Google I/O Extended` is organized by local Google community groups around the world. I/O Extended events can be run in tandem with the actual I/O event, or can be run based on the timing that is each local community feels suitable.